### ISSUE TITLE
Folders deleted from the desktop client are not getting deleted on the server

### ISSUE THREAD

**Issue by Akidxx** [owncloud/client#8393]
### Expected behaviour
Folders are not being deleted from the server after getting deleted from the desktop client

### Actual behaviour
A folder deleted from the desktop client should be deleted from the server

### Steps to reproduce
1. Connect to your owncloud account from Desktop Client
2. Delete a folder
3. Check from the owncloud server
4. The deleted folder still exist

### Server configuration

ownCloud version: 10.6.0

### Client configuration

Client version: 2.7.4
Operating system: Windows 10

The issue is not happening with files

**Comment by Akidxx** [comment]
> > * Check from the owncloud server
> > * The deleted folder still exist
> 
> Did you reload the server page? Web UI doesn't have auto refresh.

Yes I did, I also disconnect and connect again

**Comment by Akidxx** [comment]
> Can't reproduce:
> https://user-images.githubusercontent.com/214010/105855053-efc9dc80-5fe7-11eb-9600-55378a4f8014.mp4

Strange, could you try from windows ?
Here is my test from windows 10
https://share.vidyard.com/watch/WjwDCnJkqAAYyrjJkLps8c?

**Comment by TheOneRing** [comment]
If you can't reproduce this with an up to date version of Windows I'll need logs from a system where this occurres.

**Comment by Akidxx** [comment]
> If you can't reproduce this with an up to date version of Windows I'll need logs from a system where this occurres.

Same issue with an up to date windows..Also with 2.7.5 Desktop client version..Do you need logs ?

**Comment by TheOneRing** [comment]
Yes please provide logs with httplogger enabled. I can't reproduce the error.
